### PR TITLE
Add v1.16.0 release notes

### PR DESCRIPTION
**Pull request by saschagrunert**
As the title says.

### CODE REVIEW

File: src/assets/release-notes-1.16.json [hunk 1]
```diff
@@ -510,12 +550,12 @@
...
     "sigs": ["api-machinery", "auth", "cluster-lifecycle", "testing"],
     "duplicate": true,
-    "release_version": "1.16.0-alpha.2"
+    "release_version": "1.16.0-beta.0"
   },
   "78477": {
     "commit": "f31d786927a7855b3dd2200d16e2b172661fb919",
-    "text": "Add Filter extension point to the scheduling framework.",
-    "markdown": "Add Filter extension point to the scheduling framework. ([#78477](https://github.com/kubernetes/kubernetes/pull/78477), [@YoubingLi](https://github.com/YoubingLi))\n\n  Courtesy of SIG Scheduling, and SIG Testing",
+    "text": "   Add Filter extension point to the scheduling framework.",
```
Comment (justaugustus) [inline src/assets/release-notes-1.16.json]: @saschagrunert -- how did these spaces get pulled in?

Comment (saschagrunert) [reply src/assets/release-notes-1.16.json]: Via the tool, I created an follow-up PR regarding this: https://github.com/kubernetes/release/pull/870

**Comment by saschagrunert** [comment]
I think we can hold here until the final release has been done and merge it afterwards with updated notes.

/hold

**Comment by saschagrunert** [comment]
@justaugustus @jeefy this should be ready to go if nothing changes between 1.16.0-rc.2 and the final tag.

**Pull request merged**